truffle.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
Surely we don't need this file? truffle v5 only requires truffle-config.js

Reply:
ah, didn't know. Will take it out